cuda_rt_call.h

```diff
@@ -0,0 +1,17 @@
+#ifndef _CUDA_RT_CALL_H_
+#define _CUDA_RT_CALL_H_
+
+#include <cuda_runtime.h>
+
+#define CUDA_RT_CALL(call)                                                                  \
+    {                                                                                       \
+        cudaError_t cudaStatus = call;                                                      \
+        if (cudaSuccess != cudaStatus)                                                      \
+            fprintf(stderr,                                                                 \
+                    "ERROR: CUDA RT call \"%s\" in line %d of file %s failed "              \
+                    "with "                                                                 \
+                    "%s (%d).\n",                                                           \
+                    #call, __LINE__, __FILE__, cudaGetErrorString(cudaStatus), cudaStatus); \
```

Comment:
@beiwang2003 How about we add an `abort()` call here? Typically there is very little to be done in case of errors, and printing without abort quickly fills the terminal buffer.

Reply:
fixed